daemons/dss-index/app.py

```diff
@@ -45,7 +45,7 @@ def _handle_event(replica, event, context):
     executor = ThreadPoolExecutor(len(DEFAULT_BACKENDS))
     # We can't use ecxecutor as context manager because we don't want the shutdown to block
     try:
-        backend = CompositeIndexBackend(executor, DEFAULT_BACKENDS)
+        backend = CompositeIndexBackend(executor, DEFAULT_BACKENDS, context)
```

Comment:
Could you clarify what does this context refer to?

Reply:
Lambda context. It is a an object that the Amazon Lambda runtime provides. We use it frequently enough in Blue for everyone to know what `context` refers to.

A word of caution: PRs labeled as `WIP` in their title are a work in progress and could change significantly. In fact I just modified this line.

Reply:
I see. Sry I didn't notice it's a WIP PR, will be back later :)